Syntaxes/AppleScript.tmLanguage

```diff
@@ -8,7 +8,7 @@
 		<string>script editor</string>
 	</array>
 	<key>firstLineMatch</key>
-	<string>^#!.*(osascript)</string>
+	<string>^#!.*(osascript)(?!(?i:javascript)</string>
```

Comment:
Ha, this is embarassing. 😅
I’m assuming the unbalanced parens let the regex not match, which was sufficient for what I was testing at the time.

Correct, a line would contain `-l javascript`or similar.